content/applications/sales/point_of_sale/payment_methods/terminals/adyen.rst

```diff
@@ -73,12 +58,21 @@ For Odoo to know when a payment is made, you must set the terminal **Event URLs*
    `/pos_adyen/notification` in the :guilabel:`Event URLs` field; and
 #. click :guilabel:`Save` at the bottom of the screen to save the changes.
 
-Add a new payment method
-========================
+Configure the payment method
+----------------------------
+
+First, :ref:`enable the payment terminal <terminals/configuration>` and :doc:`create the related
```

Comment:
```suggestion
Start by :ref:`enabling the payment terminal <terminals/configuration>` and :doc:`creating the related
```
If you use "First", there must be a "Second" somewhere, here you only have "First" and "Finally"